scripts/common.js

```diff
@@ -1,60 +1,58 @@
 const projects = [
   {
-    name: "apisix-ingress-controller",
-    branch: "master",
-    hasChangelog: true
+    name: 'apisix-ingress-controller',
+    branch: 'master',
+    hasChangelog: true,
   },
   {
-    name: "apisix",
-    branch: "master",
-    hasChangelog: true
+    name: 'apisix',
+    branch: 'master',
+    hasChangelog: true,
   },
   {
-    name: "apisix-dashboard",
-    branch: "master",
-    hasChangelog: true
+    name: 'apisix-dashboard',
+    branch: 'master',
+    hasChangelog: true,
   },
   {
-    name: "apisix-docker",
-    branch: "master",
+    name: 'apisix-docker',
+    branch: 'master',
     hasChangelog: false,
   },
   {
-    name: "apisix-helm-chart",
-    branch: "master",
+    name: 'apisix-helm-chart',
+    branch: 'master',
     hasChangelog: false,
   },
   {
-    name: "apisix-java-plugin-runner",
-    branch: "main",
-    hasChangelog: true
+    name: 'apisix-java-plugin-runner',
+    branch: 'main',
+    hasChangelog: true,
   },
   {
-    name: "apisix-go-plugin-runner",
-    branch: "master",
-    hasChangelog: true
+    name: 'apisix-go-plugin-runner',
+    branch: 'master',
+    hasChangelog: true,
   },
   {
-    name: "apisix-python-plugin-runner",
-    branch: "master",
-    hasChangelog: true
-  }
+    name: 'apisix-python-plugin-runner',
+    branch: 'master',
+    hasChangelog: true,
+  },
 ];
 
-const languages = ["en", "zh", "es"];
+const languages = ['en', 'zh', 'es'];
```

Comment:
I think we don't need es(Spanish) for now. There was a contributor doing Spanish and he passed away about a year ago, then the process of translation is stopped, see https://github.com/apache/apisix/issues/3587#issuecomment-818397330.

I  would suggest to transfer this part as an issue and see if anyone could pick it up.

```suggestion
const languages = ['en', 'zh'];
```

Reply:
> I would suggest to transfer this part as an issue and see if anyone could pick it up.

🤣 Let's create an issue.

Reply:
> 🤣 Let's create an issue.

I've created https://github.com/apache/apisix-website/issues/1003

Reply:
Cool. Do you want to keep it as `const languages = ['en', 'zh', 'es'];` for now? I think it is better this way.

Reply:
@yzeng25 I did not change it, i just create an issue. :D

Reply:
Ok.